clr-k8s-examples/containerd-setup/containerd.conf

```diff
@@ -0,0 +1,21 @@
+[plugins]
+  [plugins.cgroups]
+    no_prometheus = false
+    [plugins.cri.containerd]
+      snapshotter = "overlayfs"
+      no_pivot = false
+      [plugins.cri.containerd.runtimes.runc]
+        runtime_type = "io.containerd.runtime.v1.linux"
+        [plugins.cri.containerd.runtimes.runc.options]
+          Runtime = "runc"
+          RuntimeRoot = "/usr/local/sbin/runc"
+      [plugins.cri.containerd.runtimes.kata-qemu]
```

Comment:
@egernst should we just use the shim-v2 version. It will allow us to skip jumping through the bash scripts as container-shim-v2 already supports passing args to the runtime binary I think.